plugins/maciVoting/contexts/CoordinatorContext.tsx

```diff
@@ -191,13 +179,64 @@ export const CoordinatorProvider = ({ children }: { children: ReactNode }) => {
     [getAuthorizationHeader, getPublicKey, publicClient]
   );
 
+  const checkMergeStatus = async (pollId: number) => {
+    const { address: pollAddress } = await getPoll({
+      maciAddress: PUBLIC_MACI_ADDRESS,
+      pollId,
+      signer,
+    });
+    const pollAbi = parseAbi(["function stateMerged() view returns (bool)"]);
```

Comment:
There is no SDK function to check `stateMerged()`? What if we use the PollFactory from the sdk? I am concern that in the future if we change the smart contract function signature, we will have to remember to change it here.

By using the PollFactory and a Poll instance we are mantaining kind of a "single source of truth"

Reply:
> There is no SDK function to check stateMerged()?

Can't see one

> By using the PollFactory and a Poll instance we are mantaining kind of a "single source of truth"

Sure, done